### ISSUE TITLE
Wrong Version, 1.8.1 starts as 1.19.2

### ISSUE THREAD

**Issue by Kartium** [FTBTeam/FTB-Modpack-Issues#5910]
### Modpack

FTB Skies Expert

### Modpack version

1.8.1

### Launcher

FTB App

### Has the pack been modified

No

### Log Files

[cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, Kartium, --version, 1.19.2-forge-43.3.8,

### Describe the bug

Me and a friend want to play on a Server together which runs on Version 1.8.1 but when we download FTB Skies Expert Version 1.8.1 it downloads it without a problem. However when we start it the version run is 1.19.2 although we first downgraded it and after that didn't work deleted and reinstalled just version 1.8.1 without having to do the 1.19.2 dowload and dowload shenanigans.
But even after this it runs version 1.19.2.

So basically my question is how do I get to run Version 1.8.1 without any problem?

### Steps to reproduce

1st. dowload FTB Skies Expert Version 1.8.1 / 2nd. run it / 3rd. it starts in the wrong version

### Expected behaviour

I expected it to run Version 1.8.1

**Comment by WyFryMU** [comment]
1.19.2 is the minecraft version, not modpack version. FTB Skies Expert uses 1.19.2 Forge.

**Comment by Kartium** [comment]
> 1.19.2 is the minecraft version, not modpack version. FTB Skies Expert uses 1.19.2 Forge.

How can I change this to 1.8.1 version
Or how do I start Skies Expert in Minecraft 1.8.1

**Comment by WyFryMU** [comment]
> > 1.19.2 is the minecraft version, not modpack version. FTB Skies Expert uses 1.19.2 Forge.
> 
> How can I change this to 1.8.1 version Or how do I start Skies Expert in Minecraft 1.8.1

You cant. 1.8.1 is the **modpack** version not the **minecraft** version. The pack plays in MC version 1.19.2 and there is no way to make it work in MC version 1.8.1.

**Comment by Gaz492** [comment]
What WyFryMU said is correct, 1.8.1 is the modpack version while 1.19.2 is the minecraft version.
If you have any other issues please open a new issue.
